datestuffs.js

```diff
@@ -0,0 +1,109 @@
+//Sam Radakovitz
+//Module 2
+
+/* exported dateStuff */
+var dateStuff = (function start() {
+
+    //private date var
+  var myDate;
```

Comment:
Rather than leave your myDate variable undefined, best practice would be to initialize it to null and check against that value when necessary.